### ISSUE TITLE
Horizontal scroll not working when the fixed top menu bar has content wider than available browser window width.

### ISSUE THREAD

**Issue by amarnath01** [bigspotteddog/ScrollToFixed#134]
Hi,
Thanks for the awesome plugin which works in almost all scenarios. However I have a corner case scenario, and would really appreciate any help on resolving this.

Problem: 
I have a top menu bar which takes up 100% width of the browser window. This has logo on one end and logout link on the other. Also this bar sometimes expands beyond the browser default width  (min width=1024px), because of the its dynamic inner contents.  The issue is that the page's horizontal browser bar does not appear when the top menu bar is bigger than the browser window. Hence I am not able to access the 'log out' and other links on the extreme ends. 

In other words.. 
if (  $('.header-wrap').width()  >=    $(window).width()  ) I need a horizontal scroll.

Also I have noticed that if I manually slightly resize the browser window, everything magically goes fine and I get the horizontal scroll. 

Also I have tried using 'dontsetWidth', ' window.resize' , 'postFixed' function handlers, however these don't seem to help.
Calling $(window).resize() has no effect. Also tried moving $(window).resize() into $(window).load.  

Anyways its a Great plugin - looking forward to communities tips, also thanks in advance for any responses/ support on the forum.
